BobsBookStore.AppHost/AppHost.cs

```diff
@@ -0,0 +1,46 @@
+#pragma warning disable CA2252
+var builder = DistributedApplication.CreateBuilder(args);
+
+// List Book
+var listBookFunction = builder.AddAWSLambdaFunction<Projects.BookInventory_Api>(name: "ListBooks",
+        lambdaHandler: "BookInventory.Api::BookInventory.Api.Functions_ListBooks_Generated::ListBooks")
+    .WithEnvironment("TABLE_NAME", "BookInventory")
```

Comment:
An option you could do would be instead of having the user manually run the CDK stacks as setup use the `builder.AddAWSCDKStack` method to instantiate the CDK stack as part of AppHost start up. You could then reference the actual DDB tables created in the stack when setting these environment variables. That would avoid hard coding the table names here.